### DISCUSSION TITLE
New sketch macro in draw.io for Confluence Cloud

### DISCUSSION THREAD

**Discussion by davidjgraph** [jgraph/drawio#2056] [General]
The [Sketch macro](https://www.diagrams.net/blog/drawio-sketch-macro) is a new feature for our integration with Confluence Cloud, delivered as part of our draw.io brand.

Any questions/suggestions please add to this discussion.

**Comment by mvillis** [comment]
Love the new sketch mode! ❤️ 
Just a heads up that unless I go incognito (or presumably clear my browser cache) there looks to be some caching gremlins that still direct me to the classic editor even when I select the Sketch Macro.

https://user-images.githubusercontent.com/3509371/120024214-d8fabd00-bfbc-11eb-85d7-8d45fbd176ba.mp4

**Reply by davidjgraph** [reply]
Even if you go to https://ac.draw.io/clear.html to clear the PWA?

**Comment by namronb** [comment]
This looks really useful!  Is there any plan to add this to draw.io for Confluence Data Center?  That's what our institution is using.  Thanks!

**Reply by davidjgraph** [reply]
Thumbs up https://github.com/jgraph/drawio/issues/2109
